_ont/ont-nokia-g-010g-t.md

```diff
@@ -62,3 +83,4 @@ The ONT has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be acc
 # Miscellaneous Links
 - [MIB file parser](https://github.com/nanomad/nokia-ont-mib-parser) for NOKIA's GPON ONTs (*helps you parsing the .mib file located in `/mnt/rwdir`*)
 - [ispreview.co.uk](https://www.ispreview.co.uk/index.php/2022/09/pictured-openreachs-future-2-5gbps-ont-for-fttp-broadband.html)
+- [nokia_ont_pwgen.py](https://github.com/amnemonic/hack-gpon) standalone version of password generator for `ONTUSER` in python.
```

Comment:
This algorithm (both the GPON one and the XGS-PON one) has been fully reverse engineered, we are choosing not to disclose it to minimize the chance of CIG changing it, this policy has been used before, see: https://github.com/hack-gpon/hack-gpon.github.io/issues/246

Reply:
Thank you for review, I have reverted commit related to adding link.